buildconfig/Jenkins/Conda/package-conda

```diff
@@ -87,9 +87,9 @@ done
 
 
 # Mamba
-setup_mamba $WORKSPACE/mambaforge "" true $PLATFORM
+setup_mamba $WORKSPACE/miniforge "" true $PLATFORM
 mamba activate base
-mamba install --yes boa conda-verify versioningit
+mamba install --yes boa conda-verify versioningit python=3.10
```

Comment:
The Python version used in the conda packages is 3.11. Do you think this needs to be Python 3.11 here as well?